<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="plot_frame">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="MpWidget" name="MpWidget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>99</y>
            <width>501</width>
            <height>371</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="save_frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QComboBox" name="pos_list">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>280</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
	background-color: rgb(45, 45, 45);
	border: 2px solid rgb(40, 40, 40);
	border-radius: 5px;
	padding: 10px;
	color: rgb(150, 150, 150);
}
QComboBox:hover {
	background-color: rgb(54, 40, 40);
	border: 2px solid rgb(150, 0, 0);
	border-radius: 5px;
	padding: 10px;
	color: rgb(150, 150, 150);
}
</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtBottom</enum>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="save_bt">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>25</y>
            <width>150</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(35, 35, 35);
	border-left: 2px solid rgb(25, 25, 25	);
	border-right: 2px solid rgb(25, 25, 25);
	border-bottom: 5px solid rgb(25, 25, 25);
	border-radius: 10px;
	color: white;
}
QPushButton:hover {
	background-color: rgb(54, 40, 40	);
	border-left: 2px solid rgb(26, 19, 19);
	border-right: 2px solid rgb(26, 19, 19);
	border-bottom: 5px solid rgb(26, 19, 19);
	border-radius: 10px;
	color: white;
}
QPushButton:pressed {
	background-color: rgb(36, 29, 29);
	border-left: 2px solid rgb(23, 16, 16);
	border-right: 2px solid rgb(23, 16, 16);
	border-top: 4px solid rgb(23, 16, 16);
	border-bottom: 1px solid rgb(23, 16, 16);
	border-radius: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>SALVAR</string>
          </property>
         </widget>
         <widget class="QPushButton" name="play_bt">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>100</y>
            <width>150</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(35, 35, 35);
	border-left: 2px solid rgb(25, 25, 25	);
	border-right: 2px solid rgb(25, 25, 25);
	border-bottom: 5px solid rgb(25, 25, 25);
	border-radius: 10px;
	color: white;
}
QPushButton:hover {
	background-color: rgb(54, 40, 40	);
	border-left: 2px solid rgb(26, 19, 19);
	border-right: 2px solid rgb(26, 19, 19);
	border-bottom: 5px solid rgb(26, 19, 19);
	border-radius: 10px;
	color: white;
}
QPushButton:pressed {
	background-color: rgb(36, 29, 29);
	border-left: 2px solid rgb(23, 16, 16);
	border-right: 2px solid rgb(23, 16, 16);
	border-top: 4px solid rgb(23, 16, 16);
	border-bottom: 1px solid rgb(23, 16, 16);
	border-radius: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>COMEÃ‡AR</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_right">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 60, 60);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="com_frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="conect_bt">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>25</y>
            <width>150</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 155, 0);
	border-left: 2px solid rgb(0, 135, 0);
	border-right: 2px solid rgb(0, 135, 0);
	border-bottom: 5px solid rgb(0, 135, 0);
	border-radius: 10px;
	color: white;
}
QPushButton:hover {
	background-color: rgb(0, 185, 0);
	border-left: 2px solid rgb(0, 135, 0);
	border-right: 2px solid rgb(0, 135, 0);
	border-bottom: 5px solid rgb(0, 135, 0);
	border-radius: 10px;
	color: white;
}
QPushButton:pressed {
	background-color: rgb(0, 160, 0);
	border-left: 2px solid rgb(0, 135, 0);
	border-right: 2px solid rgb(0, 135, 0);
	border-top: 4px solid rgb(0, 135, 0);
	border-bottom: 1px solid rgb(0, 135, 0);
	border-radius: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>CONECTAR</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="com_line">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>25</y>
            <width>150</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(45, 45, 45);
	border: 2px solid rgb(40, 40, 40);
	border-radius: 10px;
	padding: 10px;
	color: rgb(150, 150, 150);
}
QLineEdit:hover {
	background-color: rgb(54, 40, 40);
	border: 2px solid rgb(150, 0, 0);
	border-radius: 10px;
	padding: 10px;
	color: rgb(150, 150, 150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>PORTA</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="slider_frame">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>90</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(200, 71, 255);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>130</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(79, 79, 255);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>170</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(79, 255, 79);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_4">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>210</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(255, 255, 79);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_5">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>250</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(255, 127, 79);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_6">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>290</y>
            <width>300</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider{margin: 0px;}
QSlider::groove:horizontal{
	border-radius: 5px;
	height: 20px;
	margin: 0px;
	background-color: rgb(45, 45, 45);
}
QSlider::handle:horizontal{
	background-color: rgb(255, 79, 79);
	border: none;
	height: 20px;
	width: 20px;
	margin: 0px;
	border-radius: 10px;
}
QSlider::groove:horizontal:hover{background-color: rgb(54, 40, 40);}
QSlider::handle:horizontal:pressed{background-color: rgb(200, 200, 200);}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="logo_frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/logo/logos.drawio.png);
background-repeat: not-repeat;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MpWidget</class>
   <extends>QWidget</extends>
   <header>mpwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../../../../../Pictures/qt_braco.qrc"/>
 </resources>
 <connections/>
</ui>
